src/context/Theme.tsx

```diff
@@ -243,10 +243,7 @@ export const alertTextPrimary = {
     light: 'rgba(0, 0, 0, 0.8)',
     dark: 'rgb(255, 255, 255)',
 };
-export const alertBackground = {
-    light: 'white',
-    dark: semiTransparentBackgroundIntensified.dark,
-};
+export const alertBackground = paperBackground;
```

Comment:
semittransparent works elsewhere but not for notifications since they stack. Changing it to follow paper as this is easy for now.

Reply:
This is _okay_. I agree that the background of an alert should never be transparent, however an alert should be styled in a way that gives the illusion of hierarchy.

Reply:
I agree - but since the light mode one was also pure white at the top this follows that and we can make it something custom later.